core/docker/default/etc/jvm.config

```diff
@@ -1,4 +1,5 @@
 -server
+-agentpath:/usr/lib/trino/bin/libjvmkill.so
```

Comment:
Since we compile from source I believe it would work on all 3 of x64, arm64 and ppcle64 arches?

Reply:
Yes, that’s why I did it this way. I wish there was an easier way to package this so we don’t need to build every time.

Reply:
I could add a workflow in the jvmkill repo that would create releases with binaries for different architectures.

Reply:
Existing is ok for me. Also serves as an example for others who want to add similar in their own docker images.

Reply:
@nineinchnick I like that idea. Then we could just download them from the GitHub releases URL?